src/main/java/com/softserve/teamproject/service/impl/GroupServiceImpl.java

```diff
@@ -152,6 +172,32 @@ public void updateGroup(Group group, Status currentStatus, String userName)
     }
 
     groupRep.save(group);
+
+    return groupResourceAssembler.toResource(group);
+  }
+
+  /**
+   * Checks if all fields of updated group exist. If some field is not exists, we copy required
+   * fields from an old group using reflection.
+   *
+   * @param group is a group we want to update.
+   */
+  @Override
```

Comment:
Why? We have this method in the interface, so we have to mark @Override this method according to conventions.